packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -113,7 +116,7 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
             estimateSize: (_: number) => rowHeight,
             enableSmoothScroll: true,
             overscan: 3,
-            scrollingDelay: 5,
+            isScrollingResetDelay: 0,
```

Comment:
This option [was renamed](https://github.com/TanStack/virtual/discussions/714#discussioncomment-9258710).